core/classes/Events/EventHandler.php

```diff
@@ -202,16 +203,28 @@ public static function executeEvent($event, array $params = []): ?array {
     /**
      * Get a list of events to display on the StaffCP webhooks page.
      *
-     * @param bool $internal Whether to include internal events or not
      * @return array List of all currently registered events
      */
-    public static function getEvents(bool $internal = false): array {
+    public static function getEvents(): array {
```

Comment:
What are the effects of removing `$internal` here?

Reply:
The only place this is used internally is for the event list in the webhook creation page, and by modules is the websend module, which does not need internal events as well.
There should be no issues removing this